README.md

```diff
@@ -77,39 +79,83 @@ val payloadHash = "test".toByteArray().sha256()
 val signature = BCECSigner().sign(childKey.keyPair.privateKey, payloadHash)
 ```
 
-## Initializing a key from a bip32 encoded private key
+### Initializing a key from a bip32 encoded private key
 
 ```kotlin
 // Decode the base58-encoded bip32 key into an extKey.
 val extKey: ExtKey = ExtKey.deserialize("<encoded-key>".base58DecodeChecked())
 ```
 
-## Converting between key types
+## Key conversion examples
+
+### Converting a base64 encoded key into a `java.security.PublicKey` using the default (secp256k1) curve
 
 ```kotlin
+import tech.figure.hdwallet.ec.PublicKey
+import tech.figure.hdwallet.ec.extensions.toJavaECPublicKey
+import java.security.PublicKey as JavaPublicKey
+
+val encoded: String = "AmqcgLOp640tgccRYL/+PtKftP0NwHcDNzUiHNsJV+gb"
+val pubKey: JavaPublicKey = PublicKey.fromString().toJavaECPublicKey()
```

Comment:
`fromString(encoded)`?

Reply:
@mtps Good catch. Not sure how that's even compiling 🤔